jax/experimental/compilation_cache/file_system_cache.py

```diff
@@ -12,15 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Optional
 import os
+from typing import Optional
+import warnings
 
 class FileSystemCache:
 
-  def __init__(self, path: str):
+  def __init__(self, path: str, max_cache_size_bytes=32000000000):
```

Comment:
I wonder if 32 * 2**30 bytes (or 32 GiB) instead of 32 GB is better?

For example, GCP is using GiB for disk capacity

"For Compute Engine, disk size, machine type memory, and network usage are calculated in gigabytes (GB), where 1 GB is 230 bytes. This unit of measurement is also known as a gibibyte (GiB)."

https://cloud.google.com/compute/disks-image-pricing